### DISCUSSION TITLE
timeout units?

### DISCUSSION THREAD

**Discussion by IanK6449** [QuantumEntangledAndy/neolink#80] [Q&A] [answered]
Hi folks (and QuantumEntangledAndy),

I am trying to use Neolink with a Reolink Argus Eco battery powered camera.
I have noticed there is now a timeout option in the pause function which I hope will solve my problem of the battery running flat due to the RTSP feed being always on?
But, the example doesn't provide any units in the comments so I don't know whether that example timeout is in seconds or minutes or...?

Thanks either way for putting the time into this component :-)

IK

**Comment by QuantumEntangledAndy** [comment, marked as answer]
It's in seconds. Also there's a few bugs in the pause config. There are being addressed and hopefully will be merged in a few days.

**Reply by IanK6449** [reply]
Great, thanks Andy! :-)
